README.md

```diff
@@ -20,6 +20,8 @@ The NPM package for Office.js is intended as a way for you to obtain your own (n
 
 2. If you need offline access to the Office.js APIs (for example, to facilitate offline debugging).
 
+**Note**: Officially, Outlook doesn't support this. However, if you choose to proceed, you may also have to self-host a copy of Microsoft Ajax or other dependencies as needed.
```

Comment:
Why just "Outlook"?  Note of the Office Add-ins support this...

Reply:
I think "support" could mean different thing to Outlook. It is true that Microsoft Store will not accept any add-in that doesn't reference CDN directly for Office-js library. My understanding is that Outlook is saying the offline files won't work technically (which is hard for me to understand). I'll check with product team and get back.